target/arc/mmu-v6.c

```diff
@@ -415,30 +347,76 @@ arc_mmuv6_aux_set(const struct arc_aux_reg_detail *aux_reg_detail,
 
 #define ALL1_64BIT (0xffffffffffffffff)
 
+/* TODO: This is for MMU48 only. */
```

Comment:
Please be more verbose here.

Reply:
Actually, I have a zero knowledge about MMU subsystem. I just rearranged some code blocks and substituted global MMU variables by local ones for each CPU. All comments are saved as is. MMU code must be revisited a bit later.